resolwe_bio/tools/make_igv_session_archive.py

```diff
@@ -43,13 +64,16 @@ def make_xml_tree(input_file):
 
 def write_xml_file(input_file, doc):
     """Compose a name and write output file."""
-    with open(input_file.replace('temp_igv.txt', 'igv.xml'), 'wb') as f:
+    os.makedirs('IGV', exist_ok=True)
+    output_name = os.path.join('IGV', input_file.replace('temp_igv.txt', 'igv.xml'))
+    with open(output_name, 'wb') as f:
         f.write(doc)
 
 
 def main():
     """Invoke when run directly as a program."""
-    doc = make_xml_tree(args.input_file)
+    build = get_build_info(args.input_file)
```

Comment:
No need to pre-compute this and pass it to `make_xml_tree` function. Just use `genome=get_build_info(input_file)` in line 37, where you define genome.